src/PrestaShopBundle/Resources/config/services.yml

```diff
@@ -1,5 +1,7 @@
 parameters:
   multishop.settings.share_orders: !php/const Shop::SHARE_ORDER
+  ps_root_dir: !php/const _PS_ROOT_DIR_
+  ps_cache_dir: !php/const _PS_CACHE_DIR_
 
```

Comment:
No need to create parameters for now

Reply:
I removed cache dir. The root dir is useful for me to avoid creating 2 parameters for CategoriesProviderBuilder